src/flamegraph/mod.rs

```diff
@@ -711,9 +730,9 @@ fn filled_rectangle<W: Write>(
     color: Color,
     cache_rect: &mut Event<'_>,
 ) -> quick_xml::Result<usize> {
-    let x = write_usize(buffer, rect.x1);
+    let x = write!(buffer, "{:.4}%", rect.x1_pct);
```

Comment:
I wonder how much this is going to slow things down...

Reply:
I'll benchmark it to see, but what can we do about it? Integer percentages wouldn't give us enough precision.

Reply:
No, I think it's fine, I'd just be curious to see the numbers.

Reply:
Somehow this branch outperforms master by quite a bit, even though formatting the percentages does slow it down.

**master -> fluid-drawing**
```
flamegraph              time:   [17.145 ms 17.208 ms 17.283 ms]
                        thrpt:  [35.672 MiB/s 35.827 MiB/s 35.959 MiB/s]
                 change:
                        time:   [-29.290% -28.977% -28.586%] (p = 0.00 < 0.05)
                        thrpt:  [+40.028% +40.799% +41.422%]
                        Performance has improved.
```


**format percent -> write_usize**
```
flamegraph              time:   [13.489 ms 13.538 ms 13.592 ms]
                        thrpt:  [45.358 MiB/s 45.540 MiB/s 45.704 MiB/s]
                 change:
                        time:   [-20.828% -20.198% -19.557%] (p = 0.00 < 0.05)
                        thrpt:  [+24.312% +25.310% +26.307%]
                        Performance has improved.
```